<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathTemplateTab</class>
 <widget class="QWidget" name="PathTemplateTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabPathTemplate">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDesign">
      <attribute name="title">
       <string>Design</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QFrame" name="framePath">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item>
            <widget class="QSplitter" name="splitter_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QTreeWidget" name="tvPath">
              <property name="showDropIndicator" stdset="0">
               <bool>false</bool>
              </property>
              <property name="animated">
               <bool>true</bool>
              </property>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
             <widget class="QFrame" name="frmMetaBox">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="spacing">
                <number>1</number>
               </property>
               <property name="leftMargin">
                <number>1</number>
               </property>
               <property name="topMargin">
                <number>1</number>
               </property>
               <property name="rightMargin">
                <number>1</number>
               </property>
               <property name="bottomMargin">
                <number>1</number>
               </property>
               <item>
                <widget class="QLabel" name="lblMetaHeader">
                 <property name="text">
                  <string>Node Definition</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTableView" name="gridMetaBox">
                 <attribute name="verticalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QFrame" name="frameOutput">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <widget class="QTableView" name="gridOutput"/>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="btnAdd">
                <property name="text">
                 <string>Add New</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnRemove">
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnUp">
                <property name="text">
                 <string>Up</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnDown">
                <property name="text">
                 <string>Down</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>464</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCypher">
      <attribute name="title">
       <string>Cypher</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>41</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>41</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnSaveAs">
            <property name="text">
             <string>Save As...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>711</width>
              <height>18</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frmDataGrid">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
